vem/supply/ep.md

```diff
@@ -40,25 +40,25 @@ Modular Benefit Project contains:
 
 Claim contains:
 
-- Unique identifier (Id): that is issued and independent of the the MBP. The Id is used to establish a compound identifier linking the claim with its MBP and ecological project.
-- Verified Link: to the source data the claim is based off of. This contains a [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) of the data file that contains the data to be verified and either a digital signature or hash of the data file so that its integrity can be checked at anytime by comparing the data file against its verified signature or hash.
-- Date Range: the date span that the claim is being made for.
-- Environmental Effects Before: a measure of the claim before project activities.
-- Environmental Effects After: a measure of the claim after project activities.
-- Co-benefits: one or more from a list of the added benefits we get when we act to control climate change, above and beyond the direct benefits of a more stable climate.
+- Unique identifier (Id): An identifier that is issued and independent of the the MBP. The Id is used to establish a compound identifier linking the claim with its MBP and EP.
+- Verified Link: Is a reference to the source data the claim is based on. This contains a [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) pointing to the data file which can be verified. The data file should be accompanied by either a signature or a hash so that the integrity can be verified. This can be accomplished using the [W3C DID specification](https://www.w3.org/TR/did-core/).
```

Comment:
Can you change the last sentence to "For example, this could be accomplished..."

Reply:
Done